pytest_splunk_addon/splunk.py

```diff
@@ -498,7 +498,7 @@ def splunk_web_uri(splunk):
     """
     Provides a uri to the Splunk web port
     """
-    uri = f'http://{splunk["host"]}:{splunk["port_web"]}/'
+    uri = f'https://{splunk["host"]}:{splunk["port_web"]}/'
```

Comment:
We need to handle both the scenarios. `http` and `https`.  I think we have a param for http scheme already. Can you check if we can use that?

Reply:
Yes. updated to use that param.